Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3152,6 +3172,19 @@ extension BrowserViewController: PreferencesObserver {
       syncPlaylistFolders()
     case Preferences.NewTabPage.backgroundSponsoredImages.key:
       recordAdsUsageType()
+    case Preferences.Privacy.screenTimeEnabled.key:
+      if Preferences.Privacy.screenTimeEnabled.value {
+        screenTimeViewController = STWebpageController()
+        if let tab = tabManager.selectedTab {
+          recordScreentimeUsage(for: tab)
+        }
+        updateViewConstraints()
```

Comment:
Is this actually okay to call manually or do you have to call `setNeedsLayout` and `layoutIfNeeded` instead?

Reply:
let me try

Reply:
Looks like i do not even need it